### ISSUE TITLE
Reported durations are 100x too large on Linux

### ISSUE THREAD

**Issue by benbryant0** [MiniProfiler/dotnet#597]
The durations reported by MiniProfiler when run on Linux are 100x larger than they should be.

This is caused by the change [here](https://github.com/MiniProfiler/dotnet/commit/29e886ebc5a42b0c9d8737e1edb6535ae13169d6#diff-287771995cbf03416138a28c5fb05fbd8f058fd7ee653829f6001962f9c8e832L361) which was released as 4.2.43.

`Stopwatch` ticks are not the same as `DateTime` or `TimeSpan` ticks (because that's not going to confuse anyone 🙄) so using `TimeSpan.TicksPerMillisecond` to convert is not reliable. Previous versions of MiniProfiler correctly used `Stopwatch.Frequency` to account for this. See this doc page for an explanation: https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.elapsedticks?view=net-6.0#remarks

### Simple example comparing Ubuntu & Windows output:
![image](https://user-images.githubusercontent.com/30258875/150262888-cc25ded4-5556-4e5a-b1b3-cde060578f78.png)
```csharp
using StackExchange.Profiling;
using System.Diagnostics;

var stopwatch = Stopwatch.StartNew();
Thread.Sleep(1000);
stopwatch.Stop();

Console.WriteLine(@$"Running under: {System.Runtime.InteropServices.RuntimeInformation.RuntimeIdentifier}
Stopwatch:
    ElapsedTicks: {stopwatch.ElapsedTicks}
    ElapsedMilliseconds: {stopwatch.ElapsedMilliseconds}
TimeSpan.TicksPerMillisecond: {TimeSpan.TicksPerMillisecond}");

var profiler = new MiniProfiler("", new());

using (var _ = profiler.Step("")) {
    Thread.Sleep(1000);
}

profiler.Stop();

Console.WriteLine($"Profiler duration ms: {profiler.DurationMilliseconds}");
```

**Comment by NickCraver** [comment]
Dammit! I was paranoid about this and ended up the wrong way. Can you give me a sanity check on this?
```cs
internal decimal GetRoundedMilliseconds(long ticks)
{
	long times100 = ticks * 100000 / Stopwatch.Frequency;
	return times100 / 100m;
}
```
Example program:
```cs
void Main()
{
	var ticksStart = Stopwatch.GetTimestamp();
	Thread.Sleep(1000);
	var ticksEnd = Stopwatch.GetTimestamp();
	Console.WriteLine(GetRoundedMilliseconds(ticksEnd - ticksStart));
}
internal decimal GetRoundedMilliseconds(long ticks)
{
	long times100 = ticks * 100000 / Stopwatch.Frequency;
	return times100 / 100m;
}
```

**Comment by benbryant0** [comment]
Yup, seems reasonable to me!

**Comment by NickCraver** [comment]
Fixed in #602 - thanks for the help!!
